<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCE Assembly Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Open">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="actionOpenProject"/>
    </widget>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionNewProject"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="menu_Open"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionAssembly_code_editor"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionOpenProject"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Quit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/resources/icons/quit</normaloff>:/icons/resources/icons/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/open_document.png</normaloff>:/icons/resources/icons/open_document.png</iconset>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>Open existing project</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/new_document.png</normaloff>:/icons/resources/icons/new_document.png</iconset>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>Create new project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/save.png</normaloff>:/icons/resources/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/save_as.png</normaloff>:/icons/resources/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionAssembly_code_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Assembly code editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
